docs/CHANGELOG.asciidoc

```diff
@@ -44,6 +45,265 @@ Review important information about the {kib} 8.x releases.
+* Adds a severity label to SLO burn rate rule reason message {kibana-pull}158954[#158954]
+* Adds Copy JSON button to Clipboard for SLO form {kibana-pull}157902[#157902]
+* Adds Custom Metric SLI {kibana-pull}157421[#157421]
+* Adds Data Views to index pattern selector {kibana-pull}158033[#158033]
+Platform::
+* The savedObjects export API now supports exporting all types using the `*` wildcard. Please refer to the documentation {kibana-pull}159289[#159289]
+Security::
+* Hide create spaces button when limit is reached {kibana-pull}159102[#159102]
+Uptime::
+* Adds the option to send an HTML email instead of a plaintext email to users when assigned to a case {kibana-pull}159335[#159335]
+* Implement standard time formatting {kibana-pull}143799[#143799]
+* Adds a new field, called `category`, to categorize cases. Users can create a new category, set or select an existing one on a case, and filter by multiple categories on the cases table. {kibana-pull}159890[#159890]
+
+[float]
+[[fixes-v8.9.0]]
+=== Bug Fixes
+Alerting::
+* Fixes containment boundaries not being re-fetched when a query changes {kibana-pull}157408[#157408]
+* Fixes the charts on Log threshold breached details page {kibana-pull}160321[#160321]
+APM::
+* Fixes infinite loading of APM alert table {kibana-pull}161134[#161134]
+* Fixes other bucket message and sorting {kibana-pull}159919[#159919]
+* Fixes percentiles for service transaction metrics {kibana-pull}158913[#158913]
+* Fixes stack trace on errors when only available as plain text {kibana-pull}156831[#156831]
+Cases::
+* Fixes a bug in the alerts table where you cannot create a new case when attaching alerts to a case from the cases modal {kibana-pull}160526[#160526]
+* Fixes an issue where the following special characters could not be included in the case tags: `\\():<>"*` because it resulted in a bug where the case would not be displayed in the cases table when filtered for those tags. These characters are now handled correctly and the cases will be shown in the table. {kibana-pull}159815[#159815]
+Dashboard::
+* Fixes an issue where it was possible for a URL update (redirecting to the alias) to happen in the middle of building the errored state of the Dashboard {kibana-pull}161043[#161043]
+* Fixes an issue where the time slider would override custom time ranges {kibana-pull}160938[#160938]
+* Fixes an issue where if the Reset button is clicked after changing the Title or Description in Panel settings before clicking Save, the Title or Description would revert to previous values {kibana-pull}159430[#159430]
+* Fixes dashboard reset when initial state has no controls {kibana-pull}159404[#159404]
+Design::
+* Accessibility: Make links on job validation step distinguished from surrounding text {kibana-pull}160608[#160608]
+* Fixes the range slider rendering issues and performance improvements {kibana-pull}159271[#159271]
+Discover::
+* Fixes documents request missing pinned filters {kibana-pull}160693[#160693]
```

Comment:
luckily, we introduced this bug in 8.9 and also fixed it then, so this regression escaped into the wilderness

Reply:
I wanted to day: I didn't escape! 😄